content/code-review.md

```diff
@@ -9,10 +9,14 @@ consequences. To mitigate the risk of unintended consequences, teams are encoura
 principle when conducting code review before changes are merged into the codebase (e.g. implementing a pull request in git workflow).
 
 ## Projects
-- TODO
+- [Phabricator](https://github.com/phacility/phabricator/)
+- [Gerrit](https://github.com/GerritCodeReview/gerrit)
+- [GitLab](https://about.gitlab.com/stages-devops-lifecycle/code-review/)
+- [Github](https://github.com)
+- [RhodeCode](https://code.rhodecode.com/rhodecode-enterprise-ce)
+- [Review Board](https://www.reviewboard.org/)
+- [Kallithea](https://kallithea-scm.org/repos/kallithea)
 
 ## Examples
-- TODO
-
-## Links
-- TODO
+- Code review tools help ensure that ownership over code quality and security is shared.
+- Human error is a natural part of the development process, and code review tools help ensure that those errors are addressed swiftly.
```

Comment:
These are good examples of why code review tools are useful. What would be better if we gave specific examples of the actual usage of these tools which users can adopt.

Reply:
Yes, makes sense. I will add more specific examples, thanks!